### PR TITLE
Setup prettier in eslint

### PR DESCRIPTION
**Pull request by kulmann**
we might want to reduce the number of open PRs and then rebase before merging, as this affects all js and vue files.

## Description
This PR sets up the `prettier` plugin with `recommended` ruleset for eslint. As we already have lint fix running as commit hook, this automatically gets included now for files that were changed in the commit.

Applied `lint --fix` once so that we have everything with recommended code styles.

Would suggest to rebase this PR when the current sprint is done (so that the number of open PRs is smaller) and re-run the lint fixing again before merging.

## Related Issue
- Fixes https://github.com/owncloud/phoenix/issues/2992

## Motivation and Context
Prettier provides an out of the box awesome code style.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Technical debt
- [ ] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Comment by kulmann** [comment]
Open questions @LukasHirt @PVince81 
- do we want to stick with the `no semicolons` rule? Then I can add it to the prettier options
- do we want to have the number of columns as set by the default (80) or longer lines? I personally would go with something around 100-120, but have no strong preference.

**Comment by LukasHirt** [comment]
> do we want to stick with the no semicolons rule? Then I can add it to the prettier options

I'd vote for still having none

> do we want to have the number of columns as set by the default (80) or longer lines?

100 LGTM

I'd also vote for one additional rule - `vueIndentScriptAndStyle: true`

File: .eslintrc.json [hunk 1]
```diff
@@ -6,7 +6,8 @@
     },
     "extends": [
         "standard",
-        "plugin:vue/essential"
+        "plugin:vue/essential",
```
Comment (LukasHirt) [inline .eslintrc.json]: We might also want to consider switching here to recommended

**Comment by LukasHirt** [comment]
Any reason for switching to double quotes? If not, I'd prefer to keep single ones.

**Comment by kulmann** [comment]
@LukasHirt last commit is - again - applying the rules to the current code base. I'd say that we rebase to current master on friday, drop the last commit and re-run the `lint --fix` step, so that the impact is minimal.

Important: the rule `vueIndentScriptAndStyle: true` doesn't work in eslint and I didn't find a reason why. Do you have an idea? Thought it could be an issue with the indentation rules in `.editorconfig`, but removing them didn't help. 🤔

**Comment by LukasHirt** [comment]
> I'd say that we rebase to current master on friday, drop the last commit and re-run the lint --fix step, so that the impact is minimal.

Sounds good 👍 

> the rule vueIndentScriptAndStyle: true doesn't work in eslint and I didn't find a reason why

You're right. I tested it and had the same result. Strange is that most issues mentioning this are tied to IDEs and their extensions which I think wouldn't however explain why running `yarn lint` doesn't detect the missing indentation. Let's not bother then with this rule. I don't think it's worth the effort to try and debug it.

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIOCIS</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/9387/

<p>























































































































</p></details>

**Pull request merged**